tasks/_mount.yml

```diff
@@ -71,11 +72,22 @@
       # basic options
       - rw
       - _netdev
-      - noauto
+      #- noauto
       - nofail
       # rclone options
       - vfs-cache-mode={{ lin_rclone_vfs_cache_mode }}
       - dir-cache-time={{ lin_rclone_dir_cache_time }}
+      - vfs-read-chunk-size-limit=off
```

Comment:
Unfortunately I have real-life installations based on this role.
I'll have to retest if these are changed.
Of course, no size fits all.
Please add a default.yml parameter
```
lin_rclone_extra_options:
 - noauto
 ```
and flatten-merge it here.
Thus, you can set the final options to your taste.

Reply:
make it overridable on the mount level